improver_tests/ensemble_copula_coupling/test_ConvertProbabilitiesToPercentiles.py

```diff
@@ -492,6 +492,57 @@ def test_check_data_spot_forecasts(self):
         )
         self.assertArrayAlmostEqual(result.data, data, decimal=5)
 
+    def test_masked_data_below(self):
```

Comment:
Can `test_masked_data_below` and `test_masked_data_above` be combined into one test with a parameterisation for whether the data is above or below the threshold? I think the only difference between the two tests is in `set_up_probability_cube` calls.

Reply:
I'd really like to but I believe that because this test is using unittest rather than pytest parameterization isn't trivial.

Reply:
Ah of course, thanks for pointing this out.